task/backend/executor/support_test.go

```diff
@@ -131,15 +131,15 @@ func (s *fakeQueryService) FailNextQuery(forced error) {
 func (s *fakeQueryService) WaitForQueryLive(t *testing.T, script string) {
 	t.Helper()
 
-	const attempts = 10
+	const attempts = 100
```

Comment:
As seen in: https://app.circleci.com/pipelines/github/influxdata/influxdb/17190/workflows/60ed0c93-3733-492c-9c20-57a3c61f7f9b/jobs/163231